src/controllers/room.controller.js

```diff
@@ -0,0 +1,82 @@
+import { Room } from '../models/Room.models.js';
+
+export const getRooms = async (req, res) => {
+  res.send(await Room.findAll());
+};
+
+export const getOneRoom = async (req, res) => {
+  const { id } = req.params;
+  const room = await Room.findByPk(id);
+
+  if (!room) {
+    res.status(404).send({ message: 'cant get a room' });
+
+    return;
+  }
+  res.status(200);
+  res.send(room);
+};
+
+export const createRoom = async (req, res) => {
+  const { roomName } = req.body;
+
+  if (!roomName) {
+    res.status(400).send({ message: 'room name is required' });
```

Comment:
In your `createRoom` function, you are sending a response when the room name is not provided. However, you are not ending the function execution. This could lead to unexpected behavior as the function continues to execute even after a response has been sent. You should return the response to end the function execution. It would look like this: `return res.status(400).send({ message: 'room name is required' });`